src/garage/experiment/experiment_wrapper.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+# pylint: skip-file
```

Comment:
which errors does this suffer? if this is going to be in the codebase, we need to fix it.